README.md

```diff
@@ -76,6 +76,25 @@ app.use('/admin/queues', router)
 
 That's it! Now you can access the `/admin/queues` route and you will be able to monitor everything that is happening in your queues 😁
 
+### Queue options
+1. `readOnlyMode` (default: `false`)
```

Comment:
@felixmosh I really like how you set it up so each queue can have different readonly settings. =)